### ISSUE TITLE
[Enhancement] manual page - Expand "RHEL" occurrences to official name. Add description for newly added RHEL-6 and RHEL-7 profiles

### ISSUE THREAD

**Issue by iankko** [OpenSCAP/scap-security-guide#608]
The `scap-security-guide(8)` manual page in the repository (stored in `scap-security-guide/docs/scap-security-guide.8` location) currently reads:

```
RHEL6 PROFILES
       The RHEL6 SSG content...
       stig-rhel6-server-upstream
       ..
       usgcb-rhel6-server
       ..
RHEL7 PROFILES
       The RHEL7 SSG content..
       stig-rhel7-server-upstream
       ..
       rht-cpp
       ..
       common
```

There are at least two problems with the current (outdated) manual page:
- `RHEL` is not an official product name. It should be "Red Hat Enterprise Linux",
- there have been far more profiles included for RHEL/6 and RHEL/7 products, than just those currently listed in the manual page:

scap-security-guide/RHEL/6/input/profiles location:

```
$ ls -1
C2S.xml
common.xml
CS2.xml
CSCF-RHEL6-MLS.xml
desktop.xml
fisma-medium-rhel6-server.xml
ftp.xml
nist-CL-IL-AL.xml
pci-dss.xml
rht-ccp.xml
server.xml
stig-rhel6-server-upstream.xml
test.xml
usgcb-rhel6-server.xml
```

scap-security-guide/RHEL/7/input/profiles location:

```
$ ls -1
common.xml
pci-dss.xml
rht-ccp.xml
standard.xml
stig-rhel7-server-upstream.xml
test.xml
usgcb-rhel7-server.xml
```

Even if we would omit description of the `test` profile (which is not intended for production use, and serves just as a testing profile), there is still lot of profiles which are not described in the manual page (and therefore users relying on the manual page being up2date might obtain an incorrect feeling we provide just two profiles for Red Hat Enterprise Linux 6 system, and just three profiles for Red Hat Enterprise Linux 7 system - which is not the case).

Therefore the manual page should be updated to reflect the new recent profile add-ons.

**Comment by msrubar** [comment]
The issue was resolved by @redhatrises and merged by @iankko so I'm closing it.

**Comment by redhatrises** [comment]
Only part of this issue was fixed in #649 which related to changing RHEL to Red Hat Enterprise Linux. The part of this issue that needs to be fixed is adding the addition profiles with descriptions to the man page.
